src/main/java/com/yelp/clientlib/entities/options/CoordinateOptions.java

```diff
@@ -57,35 +57,35 @@ public String toString() {
     public abstract static class Builder {
 
         /**
-         * Sets latitude.
+         * @param latitude Sets latitude.
          *
          * @return this
          */
         public abstract Builder latitude(Double latitude);
 
         /**
-         * Sets longitude.
+         * @param longitude Sets longitude.
          *
          * @return this
          */
         public abstract Builder longitude(Double longitude);
 
         /**
-         * Sets accuracy of latitude, longitude.
+         * @param latitude Sets accuracy of latitude, longitude.
          *
          * @return this
          */
         public abstract Builder accuracy(Double latitude);
```

Comment:
Oops, didn't notice this before. This should be 'accuracy' instead of latitude, do you mind to also update this as well?